wpilib/wpilib/analogoutput.py

```diff
@@ -72,3 +73,5 @@ def initSendable(self, builder):
         builder.setSmartDashboardType("Analog Output")
         builder.addDoubleProperty("Value", self.getVoltage, self.setVoltage)
 
+    def getSimObject(self):
+        return AnalogOutputSim(self.channel)
```

Comment:
@virtuald did you actually review this before merging?

Reply:
... oops.